docs/README.md

```diff
@@ -1,116 +1,30 @@
-
-If you feel compelled to give further details about the app, such as it's **relationship with the VTEX admin**, don't hesitate to use this section. 
-
-## Customization
-
-The first thing that should be present in this section is the sentence below, showing users the recipe pertaining to CSS customization in apps:
-
-`In order to apply CSS customizations in this and other blocks, follow the instructions given in the recipe on [Using CSS Handles for store customization](https://vtex.io/docs/recipes/style/using-css-handles-for-store-customization).`
-
-Thereafter, you should add a single column table with the available CSS handles for the app, like the one below. Note that the Handles must be ordered alphabetically.
-
-| CSS Handles |
-| ----------- | 
-| `XXXXX` | 
-| `XXXXX` | 
-| `XXXXX` | 
-| `XXXXX` | 
-| `XXXXX` |
-
-
-If there are none, add the following sentence instead:
-
-`No CSS Handles are available yet for the app customization.`
-
-<!-- DOCS-IGNORE:start -->
-
-## Contributors (
-
-Thanks goes to these wonderful people:
-
-<!-- ALL-CONTRIBUTORS-LIST:START - Do not remove or modify this section -->
-<!-- prettier-ignore-start -->
-<!-- markdownlint-disable -->
-<!-- markdownlint-enable -->
-<!-- prettier-ignore-end -->
-<!-- ALL-CONTRIBUTORS-LIST:END -->
-
-This project follows the [all-contributors](https://github.com/all-contributors/all-contributors) specification. Contributions of any kind are welcome!
-
-<!-- DOCS-IGNORE:end -->
-
----- 
-
-Check out some documentation models that are already live: 
-- [Breadcrumb](https://github.com/vtex-apps/breadcrumb)
-- [Image](https://vtex.io/docs/components/general/vtex.store-components/image)
-- [Condition Layout](https://vtex.io/docs/components/all/vtex.condition-layout@1.1.6/)
-- [Add To Cart Button](https://vtex.io/docs/components/content-blocks/vtex.add-to-cart-button@0.9.0/)
-- [Store Form](https://vtex.io/docs/components/all/vtex.store-form@0.3.4/)
+<img width="1440" alt="Captura de Tela 2022-02-08 às 17 25 59" src="https://user-images.githubusercontent.com/80836180/153070166-e0d67a93-8bb0-4b67-8377-30d9e5049a49.png">
 
+In the store it will be possible to check the credit that is still available to be added to the gift card (this amount is calculated from the difference between the balance of the lists and the amount of this balance that has already been added to the gift card) and the current balance of the list owner's gift card.
+In addition, it will be possible to make redemptions in the "Generate Redemption" field, where the redemption value must be within the range of credit available for this redemption. Finally, the gift card code will also be present, where it can be copied by clicking on the "Copy Code" button.
```

Comment:
```suggestion
In addition, it will be possible to make redemptions in the "Generate Redemption" field, as long as the input value is within the range of credit at hand for the redemption. Finally, the gift card code will also be available and can be copied by clicking on the "Copy Code" button.
```